tests/integration/olp-cpp-sdk-authentication/AuthenticationClientTest.cpp

```diff
@@ -209,8 +210,93 @@ class AuthenticationClientTest : public ::testing::Test {
   std::shared_ptr<olp::thread::TaskScheduler> task_scheduler_;
   const std::string key_;
   const std::string secret_;
+  const std::string scope_;
 };
 
+TEST_F(AuthenticationClientTest, SignInClientScope) {
+  AuthenticationCredentials credentials("key_", secret_);
+  std::promise<AuthenticationClient::SignInClientResponse> request;
+  auto request_future = request.get_future();
+
+  EXPECT_CALL(*network_, Send(_, _, _, _, _))
+      .Times(1)
```

Comment:
How can this be with two WillOnce().
Should be Times(2) then, or remove at all as WillOnce already sets this implicitly.